test/aggregations.test.js

```diff
@@ -0,0 +1,28 @@
+var request = require('request-promise')
+var assert = require('assert')
+const config = require('config')
+
+var base_url = ('http://localhost:' + config.get('port'))
+
+describe('Aggregations response', function () {
+  this.timeout(10000)
+  var requestPath = '/api/v0.1/discovery/resources/aggregations?q=hamilton&search_scope=all'
+  // This is a bad test.
+  // * It's super dependent on our index at the time of writing.
```

Comment:
Let's talk with P when he gets back.
My uninformed opinion is that the `AggregationsSerializer` could certainly use a unit test the `packed_fields:` option. But even that wouldn't have caught this bug because this is more of an integration test....

We more or less broke this at the controller level, not the model level.